blog/_posts/2017-09-12-Final_blog_post.md

```diff
@@ -0,0 +1,44 @@
+---
+title: “”
+layout: post
+created_at: Tue Sept 12 2017
+permalink: blog/2017-09-06-final-blog-post-team-victorious-secret
+current: blog
+author: Katyayani Singh and Saumya Balodi
+twitter: VictoriuSecret
+categories:
+- student-posts
+---
+
+![Team Victorious Secret](/img/blog/2017/team_victorious_secret1.jpeg)<font color="grey"><small><i>Picture Credits: Saumya</i></small></font>
```

Comment:
Can you please use image-credits class here? Check out other blog posts for the reference.